doc/man/8/radosgw.rst

```diff
@@ -116,20 +120,20 @@ Apache 2.4, needs to be configured for use with localhost tcp.
 
 		SetEnv proxy-nokeepalive 1
 
-		ProxyPass / fcgi://127.0.01:9000/
+		ProxyPass / fcgi://localhost:9000/
+
 		</VirtualHost>
 
    CentOS/RHEL::
 
-		<VirtualHost *:80>
+		<VirtualHost {IP ADDRESS}:80>
```

Comment:
Same thing regarding "`{IP ADDRESS}`". It's ok to have the `<VirtualHost *:80>` directive use "`*`" to simplify the configuration steps.